util/topgen.py

```diff
@@ -1134,6 +1154,9 @@ def main():
 
     topname = topcfg["name"]
 
+    if not args.entropy_buffer:
+        os.remove(SRCTREE_TOP / "util/topgen/entropy_buffer.txt")
```

Comment:
Why is the buffer here removed? Is it because it can anyway be reproduced again with the seed and we just don't want it to stay around unnecessarily? If yes, I would add a comment to explain this.

Reply:
This was necessary to pass the CI. In-depth lint complains when new files are generated.

Reply:
I've added a comment to explain that the buffer is no longer needed.